index.html

```diff
@@ -4,48 +4,57 @@
     <meta charset="utf-8">
     <title>Mini Weather App</title>
     <link rel="stylesheet" href="css/style.css">
```

Comment:
Make sure your custom CSS file is loaded after any libraries you may be using (i.e. boostratp, googlefonts, flags)